src/aind_metadata_mapper/open_ephys/camstim_ephys_session.py

```diff
@@ -68,7 +67,7 @@ def __init__(self, session_id: str, json_settings: dict) -> None:
             self.recording_dir = npc_ephys.get_single_oebin_path(
                 session_inst.lims_path
             ).parent
-        except FileNotFoundError:
+        except:
```

Comment:
why was the FileNotFoundError removed? Using a bare except without specifying an exception type will catch all exceptions, including ones that you might not intend to handle (like KeyboardInterrupt, SystemExit, or MemoryError). 

This can make debugging more difficult so I'd recommend putting FileNotFoundError back or if you need to catch multiple types of errors you can do a tuple

Reply:
There was another error received here but I forget what it was. I should have written it down. I guess we can revert and if it shows up again I can commit again